services/credential-server-ui/src/pages/ConnectionPage.tsx

```diff
@@ -1,29 +1,81 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import GetQRButton from "../components/GetQRButton";
 import { config } from "../config";
-import { Divider, Grid, Typography } from "@mui/material";
+import { Box, Button, Container, Divider, Grid, Typography } from "@mui/material";
 import GetInputButton from "../components/inputOOBI/GetInputButton";
 import GetScannerButton from "../components/inputOOBI/GetScannerButton";
+import axios from "axios";
+import { Contact } from "../types.types";
+import RefreshIcon from "@mui/icons-material/Refresh";
+import { DeleteOutline } from "@mui/icons-material";
+import { UUID_REGEX } from "../constants";
 
 const ConnectionPage: React.FC = () => {
+  const [contacts, setContacts] = useState<Contact[]>([]);
+
+  useEffect(() => {
+    handleGetContacts();
+  }, []);
+
+  const handleGetContacts = async () => {
+    try {
+      setContacts(
+        (await axios.get(`${config.endpoint}${config.path.contacts}`)).data.data
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const handleDeleteContact = async (id: string) => {
+    try {
+      await axios.delete(`${config.endpoint}${config.path.deleteContact}?id=${id}`)
+      await handleGetContacts();
+    } catch (error) {
+
+    }
+  }
+
   return (
     <>
       <Typography component="h1" variant="h4" align="center">
-        Connection
+        Connect
       </Typography>
       <Grid container spacing={2} justifyContent="center">
         <Grid item xs={12} md={12}>
           <Divider />
           <GetQRButton
             name=""
             url={`${config.endpoint}${config.path.keriOobi}`}
-            onQRGenerated={() => {}}
+            onQRGenerated={() => { }}
           />
           <Divider />
           <GetScannerButton />
           <Divider />
           <GetInputButton />
           <Divider />
+          <Divider>
+            <Grid xs={12}>
+              <b>Mange connections</b><Button
```

Comment:
Manage connections should be its own box like "Connect" is above. I would prefer if they are separate